api/caasoperator/client.go

```diff
@@ -77,30 +78,30 @@ func (c *Client) SetStatus(
 
 // Charm returns information about the charm currently assigned
 // to the application.
-func (c *Client) Charm(application string) (_ *charm.URL, sha256 string, _ error) {
+func (c *Client) Charm(application string) (_ *charm.URL, force bool, sha256 string, vers int, _ error) {
```

Comment:
I think "force" as a name by itself is unclear. Could we rename to clarify? maybe a "forceUpgrade"? (here and in other places)

I think this change also warrants an update to a method comment.

Reply:
tweaked param name etc